parse_emails/parse_emails.py

```diff
@@ -32,11 +35,28 @@ def __init__(self, file_path, max_depth=3, parse_only_headers=False, file_info='
             raise Exception('Minimum max_depth is 1, the script will parse just the top email')
 
     def get_file_type(self, file_type):
+
+        mime = magic.Magic()
         if not file_type:
-            mime = magic.Magic()
             file_type = mime.from_file(self._file_path)
+
+        if file_type == 'data' or self._file_name.lower().strip().endswith('.p7m'):
```

Comment:
this shouldnt be an and here? We saw some p7m files that didnt have the same issue w signature, those would probably fail on this

Reply:
issue is now fixed after passing in the file name to the SDK constructor